docs/labs/index.rst

```diff
@@ -15,17 +15,6 @@ If we get lots of positive feedback about any one of these projects then we will
 
 .. _labs-demo:
 
-Flocker Docker plugin with CLI and GUI
```

Comment:
This page ranks, and references the labs plugin as if it is the only plugin (which is fair, it is).

Instead of being removed, it should be rewritten in such a way that this it is clear that this is not a supported confguration, that this demo is legacy, and that they can get the real plugin somewhere else.

Probably this entire section, and all the plugin references in the labs section, should carry a warning. @davidoliverSP2 what do you reckon?